tutorials/applications/QAA to solve a QUBO problem.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<p>You should change this title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/821/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>